Microsoft.Toolkit.HighPerformance/Helpers/NullableHelper.cs

```diff
@@ -0,0 +1,35 @@
+using System;
```

Comment:
The file header is missing, you need to run the `UpdateHeaders.bat` script in the `\build` folder.